nmosauth/auth_server/security_service.py

```diff
@@ -99,12 +99,11 @@ def _cleanup(self):
                 self.logger.writeInfo("mDNS stopped gracefully")
             except Exception as e:
                 self.logger.writeWarning("Could not stop mDNS gracefully: {}".format(e))
-
         self.httpServer.stop()
         self.logger.writeInfo("Stopped Http Server")
 
     def sig_handler(self):
-        self.logger.writeInfo('Pressed ctrl+c')
+        print('Pressed ctrl+c')
```

Comment:
Any reason for switching to a print? Does this match a convention from elsewhere? I'd have thought the logger set to print to the console would have the same effect.

Reply:
It appears to be a print in some of the other repos (registration, node), and its more helpful when running from the command line IMO